DiamondLightSource/SynchWeb
# LIMS-1554: Add 'State' column to proposals list page

**JIRA ticket**: [LIMS-1554](https://jira.diamond.ac.uk/browse/LIMS-1554)

**Summary**:

Users are sometimes unclear which proposal they should use, so we should make clearer which ones are closed.

**Changes**:
- Add a 'State' column to the proposals list page to display 'Open' or 'Closed'
- Add a new config variable `$closed_proposal_link`, which, if set, adds a 'Click for more info' link to the top of the main menu when a closed proposal is used.

**To test**:
- Go to the proposals list, check there is a 'State' column, with 'Open' for eg mx34263 and 'Closed' for eg mx10627
- Check clicking on the column header sorts by proposal state
- Choose a closed proposal, hover over the main menu, check a warning appears saying "This proposal is closed. You cannot create shipments, proteins or contacts."
- Add a config variable `$closed_proposal_link = 'https://www.diamond.ac.uk';`, refresh the page, check that a link for more info works and takes you to the same URL (the real page is still to be written)
- Go to an open proposal, check the warning and link are not displayed.
